apps/finance/contracts/Finance.sol

```diff
@@ -429,7 +409,7 @@ contract Finance is AragonApp, ERC677Receiver {
         return settings.periodDuration;
     }
 
-    function getBudget(address _token) transitionsPeriod public returns (uint256 budget, bool hasBudget, uint256 remainingBudget) {
+    function getBudget(address _token) transitionsPeriod public view returns (uint256 budget, bool hasBudget, uint256 remainingBudget) {
```

Comment:
This can't be `view`; `transitionsPeriod` potentially changes state.

Reply:
Oh, yes, thanks! I changed because I was having problems with some test, but reverting it now.